core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -269,6 +269,26 @@ private[spark] object Utils extends Logging {
     file.setExecutable(true, true)
   }
 
+  /**
+   * Move data to trash on truncate table given
```

Comment:
Let's remove `on truncate table` here because this is `core` module.
This can be used in another place in the future (e.g. your INSERT INTO OVERWRITE PR).
```
Move data to trash if 'spark.sql.truncate.trash.enabled' is true
```

Reply:
Sure!